.github/workflows/macos.yml

```diff
@@ -33,16 +48,36 @@ jobs:
   build_macos_release:
     name: Release build
     runs-on: macos-latest
+    env:
+      CC: ccache clang
+      CXX: ccache clang++
+      FLAGS: -O3 -DNDEBUG -pipe -march=nehalem
```

Comment:
Moving these here means we use ccache and the optimization flags to build Opus as well :-)